env.arolla.sh

```diff
@@ -149,6 +150,7 @@ else
         module use /apps/arolla/UES/jenkins/RH7.6/generic/easybuild/modules/all
         # Generated with the build script
         # implicit module purge
+        module load cmake/3.14.3
```

Comment:
this is in the COSMO_TESTENV section - for tsa with RH7.6 - module should be
cmake/3.14.5